GITHUB_ORG_MANGEMENT_POLICY.md

```diff
@@ -30,7 +30,7 @@ Owner permissions are granted will be reduced.
 The following groups are granted Ownership permissions:
 
 * TSC members
-* Chair of the Community Committee
+* Community Committee Chairperson or Director
```

Comment:
Nit: This is ambiguous. Does it mean that either the Chairperson *or* the Director gets the role? Can we split it into two lines to make it clear that both get the role?

It's a nit. You can ignore it if you want. Non-blocking. Chris Oyler is laughing as he reads this over my shoulder.

Reply:
I agree it should be listed individually.  At the very least the `or` should be `and`.

Reply:
@gibfahn assuming it is a bitwise `or`, this means both of them 😄 just kidding. I agree that these should be in separate lines to clarify that Both of them are TSC members with ownership permissions.

Reply:
Thanks for clarifying. I didn't want to take up disproportionate space. "and" was a worthy correction.